### PR TITLE
Use oc cluster for helm install

### PR DESCRIPTION
**Pull request by Nebojsa92**
`space.yaml` resource needs to have tenant value different than `""`

### CODE REVIEW

**Comment by stakater-user** [comment]
@Nebojsa92 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater-charts/application/commit/0019b26429ca0640499764d193537573ed990454/checks) has Failed!

**Comment by stakater-user** [comment]
@Nebojsa92 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater-charts/application/commit/4508d39af791e4a9503a610f6932351c07979ce9/checks) has Failed!

**Comment by stakater-user** [comment]
@Nebojsa92 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater-charts/application/commit/ed462d340bd391770edbd28db3dd943632ed8621/checks) has Failed!

**Comment by stakater-user** [comment]
@Nebojsa92 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater-charts/application/commit/82bd23991410a1941c0648bc9841601fb72053d4/checks) has Failed!

**Comment by stakater-user** [comment]
@Nebojsa92 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater-charts/application/commit/b23b935a1197cc33576e61c55969682d5ead76f1/checks) has Failed!

**Comment by stakater-user** [comment]
@Nebojsa92 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater-charts/application/commit/25eb6955641b5a8aab1586ec0ffb8200bbe3c42a/checks) has Failed!

**Comment by stakater-user** [comment]
@Nebojsa92 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater-charts/application/commit/e0c426acbdde0470d93d6bb999800db7e8a6d794/checks) has Failed!

File: .github/workflows/pull_request.yaml [hunk 1]
```diff
@@ -9,38 +9,28 @@ on:
 
 env:
   CHART_NAME: "application"
-  KUBERNETES_VERSION: "1.18.0"
-  KIND_VERSION: "0.7.0"
+  OPENSHIFT_TEST_CLUSTER: "https://api.binero-test.8sdzwd1l.kubeapp.cloud:6443"
```
Comment (rasheedamir) [inline .github/workflows/pull_request.yaml:12]: @Nebojsa92 this should be read from organization secret

Comment (Nebojsa92) [reply .github/workflows/pull_request.yaml]: fixed

**Comment by stakater-user** [comment]
@Nebojsa92 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater-charts/application/commit/b5713db073d4c67e8d0a4955359addc42e307bf5/checks) has Failed!

**Comment by stakater-user** [comment]
@hanzala1234 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater-charts/application/commit/4a2ff67af17531d178a0f9dc6f299d26acc41997/checks) has Failed!

**Pull request closed without merging**